packages/electron-updater/src/windowsExecutableCodeSignatureVerifier.ts

```diff
@@ -35,8 +35,7 @@ export function verifySignature(publisherNames: Array<string>, tempUpdateFile: s
       }
       catch (e) {
         logger.warn(`Cannot execute Get-AuthenticodeSignature: ${error}. Ignoring signature validation due to unknown error.`)
-        resolve(null)
-        return
+        throw error
```

Comment:
@Mik317 can you take care to replace `throw error` with `return`?

Reply:
Hi @toufik-airane :)
I've updated on my `master` branch as you suggested previously :).
I'm not sure if it gets reflected directly here or I need to do something else, please let me know in case :)

Regards,
Mik

Reply:
@Mik317 - I have already added these changes in your commit in this pull request for you - nothing for you to do! 🍰